### ISSUE TITLE
Request : help to modify documentation files

### ISSUE THREAD

**Issue by schmurtzm** [Stef-aap/RFLink-ESP#4]
Hi,

Thanks for this awesome work.
I started a fork with some modifications on the mqtt messages formatting.
Would you accept to share the original OneNote file to help the modification of the documentation ?

Thank you.
Best regards.

**Comment by Stef-aap** [comment]
hello ?
Yes,I'm willing to give you the original onenote files. Can you handle Onenote 2007 files ?
cheers, Stef

**Comment by schmurtzm** [comment]
Than you!
Yes, no problem with this format !

**Comment by schmurtzm** [comment]
Thanks a lot Stef-aap !
For information you will find [my modified source here](https://github.com/schmurtzm/RFLink-ESP).
I have added a web interface to configure wifi and MQTT and a home page with the last events received.
Your generic protocol works pretty well with my different remotes at home :)
FYI, there's [a french topic on the arduino's forum](https://forum.arduino.cc/index.php?topic=615053.0) which talk essentially about your source code :)

One last thing that I learned on this french topic :  [a source of RFlink-ESP by couin3](https://github.com/couin3/RFLink) that keep compatibility with the original RFlink R33 plugins. I like your generic protocol : it's easier to make it work with a lot of devices but may be couin3's code could be a good way to support more protocols and his fetchsignal() function has been [greetly improved](https://translate.google.fr/translate?hl=fr&sl=auto&tl=en&u=https%3A%2F%2Fforum.arduino.cc%2Findex.php%3Ftopic%3D615053.msg4308697%23msg4308697).
